testci/install.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
```

Comment:
Does this still work with out requirements.txt file? That's what CircleCI used to read.